chat/src/main/java/com/qiscus/sdk/data/remote/QiscusApi.java

```diff
@@ -302,6 +302,12 @@ public static QiscusApi getInstance() {
                 .toList();
     }
 
+    public Observable<List<QiscusChatRoom>> getRoomsInfo(List<String> roomId, List<String> uniqueId) {
```

Comment:
ganti list of integer aja ya @adicatur soalnya room id itu int di pojonya. List roomIds

Reply:
sekalian ya kalo bentuknya list, namanya pakai jamak ya. roomIds, uniquesIds